cli/common/utils.go

```diff
@@ -564,6 +564,36 @@ func ParseDependencies(rawDeps []string) (PackDependencies, error) {
 	return deps, nil
 }
 
+func isCommandAvailable(command string) bool {
+	commandName := strings.Split(command, " ")[0]
+
+	_, err := exec.LookPath(commandName)
```

Comment:
You are trying to check if a specified command exists on your local machine, but the script will be run on the remote host where the package is installed.